### PR TITLE
Add INSERT_NOSIM and REMOVE_NOSIM jobs

### PR DESCRIPTION
**Pull request by sondreso**
Resolves https://github.com/equinor/semeio/pull/113

### CODE REVIEW

**Comment by sondreso** [comment]
Inputs on how this could be tested is appreciated :slightly_smiling_face:

**Comment by oyvindeide** [comment]
A really ugly integration test by executing it with `libres`? 🤷

**Comment by sondreso** [comment]
I was hoping to avoid that, but it might be to much to ask :see_no_evil:

**Comment by oyvindeide** [comment]
We don´t have anything for the other `JOB_DESCRIPTION` files, and these are not special in that regards. If you come up with something that works reasonably well we should implement it on the others as well. That being said we should find another way to do this asap.

**Comment by sondreso** [comment]
The easiest is maybe to crate a ERT setup which includes all of the jobs, and then run that as one very big integration test?

**Comment by oyvindeide** [comment]
We should perhaps have some test fixture for this, resolving the job, preferably in ert, but guess it has to be libres. It is one of the most important APIs we have.

**Comment by sondreso** [comment]
Updated with integration tests

File: tests/jobs/nosim/test_nosim.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    ):
+        p = subprocess.Popen(
+            ["ert", "test_run", "nosim.ert", "--verbose"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=os.environ.copy(),
+        )
+        p.wait()
+
+    with open("nosim/real_0/iter_0/TEST.DATA") as fh:
```
Comment (oyvindeide) [inline tests/jobs/nosim/test_nosim.py:69]: I can´t get this to pass locally on my machine, any idea why? Running osx, example output:
`fh.read()`:
```
RUNSPEC
NOSIM
-- and more RUNSPEC
```
Expected:
```
RUNSPEC
-- and more RUNSPEC
```

Comment (sondreso) [reply tests/jobs/nosim/test_nosim.py:69]: Have you tried running the sed command manually to see if you get the expected results? There are slight differences in how sed works on mac and linux, but I'm surprised if it's this big of a difference!

Comment (sondreso) [reply tests/jobs/nosim/test_nosim.py:69]: Apparently:
> . -i works with GNU sed, recent versions of NetBSD, OpenBSD sed, but not OS X's sed. 

https://unix.stackexchange.com/a/131940

Comment (sondreso) [reply tests/jobs/nosim/test_nosim.py:69]: Do we care about this? It's an copy of the current NOSIM job, so neither will work on OSX.

Comment (oyvindeide) [reply tests/jobs/nosim/test_nosim.py:69]: Looks like the alternative is some funky rewrite, I support just keeping it as is.

**Pull request merged**